dlp/api/DlpTest/DlpTest.cs

```diff
@@ -305,19 +308,47 @@ public void TestKMap()
             Assert.Matches(new Regex("Values: \\[\\d{2},Female,US\\]"), output.Stdout);
         }
 
-        [Fact(Skip = "https://github.com/GoogleCloudPlatform/dotnet-docs-samples/issues/510")]
+        [Fact]
         public void TestJobs()
         {
+            // Create job.
+            DlpServiceClient dlp = DlpServiceClient.Create();
+            var dlpJob = dlp.CreateDlpJob(new CreateDlpJobRequest()
+            {
+                ParentAsProjectName = new ProjectName(ProjectId),
+                RiskJob = new RiskAnalysisJobConfig()
+                {
+                    PrivacyMetric = new PrivacyMetric()
+                    {
+                        CategoricalStatsConfig = new PrivacyMetric.Types.CategoricalStatsConfig()
+                        {
+                            Field = new FieldId()
+                            {
+                                Name = "zip_code"
+                            }
+                        }
+                    },
+                    SourceTable = new BigQueryTable()
+                    {
+                        ProjectId = "bigquery-public-data",
+                        DatasetId = "san_francisco",
+                        TableId = "bikeshare_trips"
+                    }
+                }
+            });
             Regex dlpJobRegex = new Regex("projects/.*/dlpJobs/r-\\d+");
 
-            // List
-            ConsoleOutput listOutput = _dlp.Run("listJobs", CallingProjectId, "state=DONE", "RiskAnalysisJob");
-            Assert.Matches(dlpJobRegex, listOutput.Stdout);
-
-            // Delete
-            string jobName = dlpJobRegex.Match(listOutput.Stdout).Value;
-            ConsoleOutput deleteOutput = _dlp.Run("deleteJob", jobName);
-            Assert.Contains($"Successfully deleted job {jobName}", deleteOutput.Stdout);
+            _retryRobot.Eventually(() =>
+            {
+                // List jobs.
+                ConsoleOutput listOutput = _dlp.Run("listJobs", CallingProjectId, "state=DONE", "RiskAnalysisJob");
+                Assert.Matches(dlpJobRegex, listOutput.Stdout);
+
+                // Delete created job.
```

Comment:
Clean up code should go in a Dispose() method of this class.  Otherwise, if line 345's assertion fails, the code will leak resources.

Reply:
I'll add a dispose method that deletes any DLP jobs found, but we still want this delete check here because it's actually testing the `deleteJob` command.